tests/Fixer/Operator/PreIncrementFixerTest.php

```diff
@@ -27,6 +27,7 @@
      * @param string      $expected
      * @param null|string $input
      *
+     * @group legacy
```

Comment:
btw, we use it only with combination of deprecated code being tested

Reply:
of course, fixed